core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaSchemaRegistry.java

```diff
@@ -189,21 +192,31 @@ private void registerProviders(
 
   protected KafkaStore<SchemaRegistryKey, SchemaRegistryValue> kafkaStore(
       SchemaRegistryConfig config) throws SchemaRegistryException {
+    Map<String, Object> handlerConfigs =
+        config.originalsWithPrefix(SchemaRegistryConfig.KAFKASTORE_UPDATE_HANDLERS_CONFIG);
+    handlerConfigs.put(StoreUpdateHandler.SCHEMA_REGISTRY, this);
+    List<SchemaUpdateHandler> customSchemaHandlers =
+        config.getConfiguredInstances(SchemaRegistryConfig.KAFKASTORE_UPDATE_HANDLERS_CONFIG,
```

Comment:
Does this return an empty list if no update_handler_configs is specified?

Reply:
Yes, thanks for asking